app/grants/templates/grants/cart-vue.html

```diff
@@ -45,169 +46,426 @@
+            <div v-if="grantsCountByTenant.ETH !== undefined" class="bg-white border-grey border-top-2 bottom-0 px-2 py-4 position-sticky text-center w-100 d-block d-md-none">
+              <eth-checkout-button 
+                :standard-checkout="standardCheckout"
+                :is-zk-sync-down="isZkSyncDown"
+                :is-polygon-down="isPolygonDown"
+                :current-tokens="currentTokens"
+                :donation-inputs="donationInputs"
+                :max-cart-items="maxCartItems"
+                :grants-by-tenant="grantsByTenant"
+                :grants-under-minimal-contribution="grantsUnderMinimalContribution"
+                :network="network"
+                >
+              </eth-checkout-button>
             </div>
-            {% include 'shared/create_collection_modal.html' %}
           </div>
         </grants-cart>
       </div>
-      {% include 'shared/footer.html' %}
+    {% include 'shared/footer.html' %}
     </div>
 
     {% include 'shared/current_profile.html' %}
     {% include 'shared/footer_scripts.html' with slim=1 %}
     {% include 'shared/fingerprintjs.html' %}
+    <script id="eth-checkout-template" type="text/x-template">
+      <b-dropdown variant="primary"
+        block
+        no-caret
+        toggle-class="text-decoration-none"
+        class="rounded dropdown-checkout btn btn-primary p-0 w-100"
+        text="I'm ready to checkout"
+      >
+        <template v-slot:button-content>
+          <span class="text-primary">
+            I'm Ready to Checkout
+          </span>
+        </template>
+        <b-dropdown-item-button>
+          <div>
+            <button class="btn p-0 checkout-logo" id='js-fundGrants-button'
+              @click="standardCheckout" :disabled="isCheckoutOngoing || grantsUnderMinimalContribution.length > 0"
+            >
+    
+              <svg width="13" height="20" viewBox="0 0 13 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M12.7231 10.1875L6.36364 13.8125L0 10.1875L6.36364 0L12.7231 10.1875ZM6.36364 14.9766L0 11.3516L6.36364 20L12.7273 11.3516L6.36364 14.9766Z" fill="#0E0333"/>
+              </svg>
+    
+              Standard Checkout
+            </button>
+          </div>
+        </b-dropdown-item-button>
+    
+        <!-- CHECKOUT WITH POLYGON -->
+        <b-dropdown-item-button v-if="!isPolygonDown">
+          <grants-cart-ethereum-polygon
+            inline-template
+            :current-tokens="currentTokens"
+            :donation-inputs="donationInputs"
+            :max-cart-items="100"
+            :grants-by-tenant="grantsByTenant"
+            :grants-under-minimal-contribution="grantsUnderMinimalContribution"
+          >
+            <div>
+              <!-- Checkout with Polygon button -->
+              <button
+                @click="checkoutWithPolygon()"
+                class="btn p-0 checkout-logo"
+                :disabled="grantsUnderMinimalContribution?.length > 0"
```

Comment:
@chibie @thelostone-mc @gdixon @ksolo 

I have altered the condition in this disabled attribute (I have moved some verifications to the click handler).
I would also move this verification to the click handler and display an error message if verification fails.

Is there any good reason why not to do this? I mean yes, we'll have to be careful and test the entire flow again, but is there anything particular to consider?
As I have understood from @PixelantDesign yesterday, there where some considerations in the past.

Reply:
I think this should be fine, as you say, let's get it tested thoroughly over on staging before we push it out to prod 🙌